project_task_code/migrations/11.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# copyright 2020 Bogdan Stanciu
```

Comment:
Could you please add the copyright information like in https://github.com/OCA/social/blob/12.0/mail_activity_done/migrations/12.0.2.0.0/post-migration.py, OCA style :-)

```
# -*- coding: utf-8 -*-
# Copyright 2020 Bogdan Stanciu <https://www.mysite.com>
# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```